Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -66,105 +113,115 @@ internal void InstallOrUpdate(UserPlugin plugin)
+                    Context.API.GetTranslation("plugin_pluginsmanager_download_success"));
             }
             catch (Exception e)
             {
                 Context.API.ShowMsg(Context.API.GetTranslation("plugin_pluginsmanager_downloading_plugin"),
-                                Context.API.GetTranslation("plugin_pluginsmanager_download_success"));
+                    Context.API.GetTranslation("plugin_pluginsmanager_download_success"));
 
                 Log.Exception("PluginsManager", "An error occured while downloading plugin", e, "PluginDownload");
             }
 
-            Application.Current.Dispatcher.Invoke(() => { Install(plugin, filePath); Context.API.RestartApp(); });
+            Application.Current.Dispatcher.Invoke(() =>
+            {
+                Install(plugin, filePath);
+                Context.API.RestartApp();
+            });
         }
 
         internal List<Result> RequestUpdate(string search)
         {
             var autocompletedResults = AutoCompleteReturnAllResults(search,
-                                                                    Settings.HotkeyUpdate,
-                                                                    "Update",
-                                                                    "Select a plugin to update");
+                Settings.HotkeyUpdate,
+                "Update",
+                "Select a plugin to update");
 
             if (autocompletedResults.Any())
                 return autocompletedResults;
 
             var uninstallSearch = search.Replace(Settings.HotkeyUpdate, string.Empty).TrimStart();
 
 
-            var resultsForUpdate = 
-                    from existingPlugin in Context.API.GetAllPlugins()
-                    join pluginFromManifest in pluginsManifest.UserPlugins
-                        on existingPlugin.Metadata.ID equals pluginFromManifest.ID
-                    where existingPlugin.Metadata.Version != pluginFromManifest.Version
-                    select 
-                      new 
-                      {
-                          pluginFromManifest.Name, 
-                          pluginFromManifest.Author, 
-                          CurrentVersion = existingPlugin.Metadata.Version, 
-                          NewVersion = pluginFromManifest.Version,
-                          existingPlugin.Metadata.IcoPath,
-                          PluginExistingMetadata = existingPlugin.Metadata,
-                          PluginNewUserPlugin = pluginFromManifest
-                      };
+            var resultsForUpdate =
+                from existingPlugin in Context.API.GetAllPlugins()
+                join pluginFromManifest in pluginsManifest.UserPlugins
+                    on existingPlugin.Metadata.ID equals pluginFromManifest.ID
+                where existingPlugin.Metadata.Version.CompareTo(pluginFromManifest.Version) > 0
```

Comment:
just going to make a change here, dont use compareto, use equals or ==

use compareto for sorting

Reply:
Shouldn't we only update plugin if the version is higher on manifest instead of they are not equal?
Sorry, the comparison should be less than 0.😖

Reply:
my apologies, you are right actually. I was coming from the perception that we dont bother actually compare the version strings, since new updates should have version greater than the previous, but using compareto is better code.

I have changed it to less than 0

Reply:
Haha thank you. I found that due to my computer has an unpublished window walker, which has version higher than the one in our manifest. 😝